pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -105,12 +115,13 @@ export const HandlerForm: React.FC<HandlerFormProps> = ({ handlerId, endpointId
     // get the inputs and check if set other set null
     let body: {} = {
```

Comment:
Because body is currently of type `{}` it does not have (e.g.) `.conditions` which is giving a Typescript error on line `143` so body should be of type `any`.

Reply:
Edit: solved.